index.js

```diff
@@ -0,0 +1,108 @@
+const currentDay = document.querySelector(".current-day");
+const currentDate = document.querySelector(".current-date");
+const currentMonth = document.querySelector(".current-month");
+const currentYear = document.querySelector(".current-year");
+const tbody = document.querySelector("tbody");
+const leftChevron = document.querySelector(".left-chevron i");
+const rightChevron = document.querySelector(".right-chevron i");
+
+const currDate = new Date();
+let moveDate = new Date();
+
+const week = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+const month = [
+  "JAN",
+  "FEB",
+  "MAR",
+  "APR",
+  "MAY",
+  "JUN",
+  "JUL",
+  "AUG",
+  "SEP",
+  "OCT",
+  "NOV",
+  "DEC",
+];
+
+function printDayContainer(currDay) {
+  currentDay.innerHTML = `${week[currDay.getDay()]}`;
+  currentDate.innerHTML = `${currDay.getDate()}`;
+  currentMonth.innerHTML = `${month[currDay.getMonth()]}`;
+  currentYear.innerHTML = `${currDay.getFullYear()}`;
+}
+
+function printCalendar(currDay) {
+  let dayIdx = 1;
+  const firstDate = new Date(currDay.getFullYear(), currDay.getMonth(), 1);
+  const lastDate = new Date(currDay.getFullYear(), currDay.getMonth() + 1, 0);
+
+  for (var i = 0; i < 5; i++) {
+    const tr = document.createElement("tr");
+    tbody.appendChild(tr);
+    for (var j = 0; j < week.length; j++) {
+      const td = document.createElement("td");
+      tr.appendChild(td);
+
+      if (
+        (dayIdx > 1 && dayIdx <= lastDate.getDate()) ||
+        (i === 0 && j === firstDate.getDay())
+      ) {
+        td.innerHTML = dayIdx;
+        if (
+          dayIdx === currDay.getDate() &&
+          currDay.getFullYear() === currDate.getFullYear() &&
+          currDay.getMonth() === currDate.getMonth()
```

Comment:
if 내에 많은 조건이 들어갔을 경우 하나의 변수로 선언해주어 가독성을 높일 수도 있습니다.
```suggestion
        const isMatchedToday =
          dayIdx === currDay.getDate() &&
          currDay.getFullYear() === currDate.getFullYear() &&
          currDay.getMonth() === currDate.getMonth();
         if (isMatchedToday) {
           td.style.color = "red";
         }
```